tests/data/pnc/outbreak-confirmation/arborator.config.json

```diff
@@ -0,0 +1,80 @@
+        "count_geo_loc_name_state_province_territory_AB": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_AB","display":"True"},
+        "count_geo_loc_name_state_province_territory_BC": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_BC","display":"True"},
+        "count_geo_loc_name_state_province_territory_MB": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_MB","display":"True"},
+        "count_geo_loc_name_state_province_territory_NB": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_NB","display":"True"},
+        "count_geo_loc_name_state_province_territory_NL": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_NL","display":"True"},
+        "count_geo_loc_name_state_province_territory_NS": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_NS","display":"True"},
+        "count_geo_loc_name_state_province_territory_NT": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_NT","display":"True"},
+        "count_geo_loc_name_state_province_territory_NU": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_NU","display":"True"},
+        "count_geo_loc_name_state_province_territory_ON": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_ON","display":"True"},
+        "count_geo_loc_name_state_province_territory_PE": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_PE","display":"True"},
+        "count_geo_loc_name_state_province_territory_QC": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_QC","display":"True"},
+        "count_geo_loc_name_state_province_territory_SK": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_SK","display":"True"},
+        "count_geo_loc_name_state_province_territory_YK": { "data_type": "None","label":"count_geo_loc_name_state_province_territory_YK","display":"True"}
+    },
+
+    "linelist_columns":{
+        "sample_name": { "data_type": "Categorical","label":"sample_name","display":"True", "default": ""},
+        "new_isolates_date": { "data_type": "min_max","label":"new_isolates_date","display":"True", "default": ""},
+        "gas_denovo_cluster_address": { "data_type": "Categorical","label":"gas_denovo_cluster_address","display":"True", "default": ""},
+        "provisional_outbreak_code": { "data_type": "None","label":"provisional_outbreak_code","display":"True", "default": ""},
+        "national_outbreak_code": { "data_type": "None","label":"national_outbreak_code","display":"True", "default": ""},
+        "national_outbreak_status": { "data_type": "None","label":"national_outbreak_status","display":"True", "default": ""},
+        "national_outbreak_date_assigned": { "data_type": "min_max","label":"national_outbreak_code_date_assigned","display":"True", "default": ""},
```

Comment:
The source metadata `national_outbreak_date_assigned` and label `national_outbreak_code_date_assigned` should both be consistent.